WhoIsThat/WhoIsThat/HomePage.xaml.cs

```diff
@@ -9,12 +12,52 @@
 
 namespace WhoIsThat
```

Comment:
Namespace should probably be more descriptive, for example `WhoIsThat.Views`

Reply:
Strangely enough did not find a way in VS to change namespace names according to folders or how to set them up hierarchically.